### PR TITLE
[Feature] 382800 - Sending Attachments

### PR DESCRIPTION
**Pull request by mpamaro**
This feature fixes various widgets and add some new ones. See the complete change list below:

- Add widget to get the file extension icon
- Fix toast size and margin
- Fix spinner loading background color
- Change loading widget used on icon button
- Add a few properties to send button
- Fix loading on image message bubble
- Add disable layout to text form field
- Add "#" to ticket number on DSTicketMessage
- Add sequential suffix to ticket number in DSCard, so it can be passed through ticket message
- Improve readability on group card

### CODE REVIEW

File: lib/src/widgets/buttons/ds_icon_button.widget.dart [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import 'package:flutter/material.dart';
 
-import '../../themes/colors/ds_colors.theme.dart';
-import '../animations/ds_fading_circle_loading.widget.dart';
+import '../../../blip_ds.dart';
```
Comment (leonardogbr) [inline lib/src/widgets/buttons/ds_icon_button.widget.dart:3]: Fix to import only needed files.

File: lib/src/controllers/chat/ds_file_message_bubble.controller.dart [hunk 2]
```diff
@@ -1,71 +1,15 @@
 import 'package:filesize/filesize.dart';
-import 'package:flutter/material.dart';
 import 'package:get/get.dart';
-import 'package:path/path.dart' as path_utils;
 
 import 'package:blip_ds/src/services/ds_file.service.dart';
```
Comment (leonardogbr) [inline lib/src/controllers/chat/ds_file_message_bubble.controller.dart:4]: Use the relative path instead of absolute path

**Review by leonardogbr — CHANGES_REQUESTED** [review]

File: lib/src/controllers/chat/ds_file_message_bubble.controller.dart (hunk 2 above)
Comment (mpamaro) [reply lib/src/controllers/chat/ds_file_message_bubble.controller.dart:4]: Done

File: lib/src/widgets/buttons/ds_icon_button.widget.dart (hunk 1 above)
Comment (mpamaro) [reply lib/src/widgets/buttons/ds_icon_button.widget.dart]: Done

**Review by leonardogbr — APPROVED** [review]

**Review by PaollaLira — APPROVED** [review]

**Pull request merged**